CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [0.3.0-1]
+
+Pulled upstream rust-vmm/kvm-bindings v0.3.0.
+
+### Added
+
+- Enabled `fam-wrappers` support on arm and arm64.
```

Comment:
Is this done upstream or here?

Reply:
I understand this is done here since the pull from upstream, is above. The logic here would be to record only our changes and mention pulls from upstream. 
So this kind of looks like a merge of upstream changelog. @dianpopa  Is this correct ?

Reply:
These are the detailed changes **pulled from upstream**.

Reply:
> Is this done upstream or here?

This is done upstream, associated PR is this https://github.com/rust-vmm/kvm-bindings/pull/22.

Reply:
Got it, thanks. I think it would be a bit clearer if in the `Added` and `Changed` sections we write only what diverges from upstream.

Reply:
> Got it, thanks. I think it would be a bit clearer if in the `Added` and `Changed` sections we write only what diverges from upstream.

But then, how do you know what the new release changes? You need to check upstream to understand what changed locally..

I say we keep the local changelog complete with all changes. *Improvement idea*: tag entries in changelog as local or upstream - either as separate sections or using a tag (example `upstream`) for each (upstream) line.

Reply:
Example scenario:

`entry 1,2,3` were added **in upstream** between `v0.2.0` and `v0.3.0`: 
In our fork between `v0.2.0-1` and `v0.3.0-1`: `entry 1,2,3` (coming from upstream) and `custom Z and W` were added.

```
# Our Changelog

## [v0.3.0-1]

### Added
- [upstream] upstream v0.3.0 changelog entry 1
- [upstream] upstream v0.3.0 changelog entry 2
- [upstream] upstream v0.3.0 changelog entry 3
- added custom change Z
- added custom change W

## [v0.2.0-1]

### Added
- added custom change X

### Changed
- changed api call Y

## [v0.2.0]

Forked upstream kvm-bindings v0.2.0.
```

Or `[from upstream]` instead of `[upstream]` or smth else 😄

Reply:
@acatangiu  take a look now and let me know